e2eAIOK/ModelAdapter/src/engine_core/adapter/adversarial/DA_Loss.py

```diff
@@ -120,7 +119,7 @@ def forward(self, f_s: torch.Tensor, f_t: torch.Tensor,
 class CACDomainAdversarialLoss(nn.Module):
     def __init__(self, **kwargs):
         super().__init__()
-        self.loss_weights = kwargs.pop("loss_weights")
+        self.loss_weights = kwargs.pop("loss_weight")
```

Comment:
I saw in input args, it is still loss_weights? check "run_dist.sh":line 40, is changing to 'loss_weight' a correct fix?

Reply:
Yes, it is right. There are just happened to be the same name, but turned out different things, inside the python code, it is called `lose weight`.

Reply:
Thanks